src/Secure/OpenIdConnectAuthentication.php

```diff
@@ -48,8 +48,9 @@ class OpenIdConnectAuthentication extends AuthenticatedConnection
      * Please note that for most calls an office is mandatory. If you do not supply it
      * you have to pass it with every request, or call setOffice.
      */
-    public function __construct(OAuthProvider $provider, string $refreshToken, ?Office $office)
+    public function __construct(OAuthProvider $provider, string $refreshToken, ?Office $office, string $accessToken = null)
```

Comment:
Shouldn't this be `?string $accessToken` ?
```suggestion
    public function __construct(OAuthProvider $provider, string $refreshToken, ?Office $office, ?string $accessToken = null)
```